release/bin/macos/rttr.command

```diff
@@ -23,6 +23,8 @@ chmod 0755 ../rttr.command ../bin/RTTR/s25update ../bin/s25client ../bin/RTTR/so
 RTTR_TEST_FILEA="share/s25rttr/S2/DATA/CREDITS.LST"
 RTTR_TEST_FILEB="share/s25rttr/S2/GFX/PALETTE/PAL5.BBM"
 
+unset LANG
```

Comment:
Why? I was rather thinking about supporting that additionally in mygettext

Reply:
well, i tried everything to remove that error caused by the entry (with and without UTF-8) it always generated the error "locale::facet::_S_create_c_locale name not valid"

Reply:
IMO this could only be caused by wrong locale settings (e.g. var set wrong, lang not installed)
See https://stackoverflow.com/questions/19100708/mongodb-mongorestore-failure-localefacet-s-create-c-locale-name-not-valid/32762296#32762296 and https://stackoverflow.com/questions/19100708/mongodb-mongorestore-failure-localefacet-s-create-c-locale-name-not-valid

We should not work-around user bugs... Unsetting `LANG` would influence the whole system-language setting and the user might end up with a wrong language in rttr

Reply:
yea but that only happens on osx, where you dont have the possibility to install "locales" or something like that.

Reply:
I found a couple other options that I like much better:

1. Use libc++ instead of libstdc++ https://stackoverflow.com/questions/11190072/setting-locales-on-os-x-crashes
2. Build gcc with `--enable-clocale` https://www.thecodingforums.com/threads/std-locale-problem-on-mac-osx.752939/
3. Don't use `std::locale("")` which boost also does for OSX and related. (Based on https://svn.boost.org/trac10/ticket/1458 and current boost src)

While it would be great to know if 1. or 2. works I'd go for 3 and remove the `unset LANG` as I'm sure keeping it would make our "System-default" language setting fail.

Reply:
I'll test the "--enable-clocale" first - for libc++ I have to rebuild the boost library too

Reply:
`--enable-clocale` seems to have no effect

Reply:
Added a new commit which should fix this -.-